src/Components/Progress.js

```diff
@@ -1,3 +1,15 @@
-export default function Progress() {
-  return null;
+export default function Progress({allDonations, target}) {
+console.log("all donations", allDonations);
+let donated = 0;
+for(let donate of allDonations.amount){
+  donated += donate
+}
+
+
+  return <section className="progress">
+  <h2>
+    Raised <span className="secondary">$0</span> of
```

Comment:
You have `0` here. What variable would take its place to show what has been donated?